### DISCUSSION TITLE
11 Preview 5 Release?

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [AvaloniaUI/Avalonia#10101] [Q&A] [answered]
It's been quite a while since we've had a new preview release for 11. Is there a time frame in which we can expect a new release?

**Comment by robloo** [comment]
Duplicate: https://github.com/AvaloniaUI/Avalonia/discussions/9920

**Reply by wj8400684** [reply]
> 重复：#9920

This has been three weeks, and preview 5 has not been released yet!

**Reply by robloo** [reply]
There is a lot going on and it's an open source project. I'm sure it will be released soon and we won't worry about a few weeks here and there by the time 11.0 releases. You can always run with nightly builds.

**Reply by wj8400684** [reply]
> There is a lot going on and it's an open source project. I'm sure it will be released soon and we won't worry about a few weeks here and there by the time 11.0 releases. You can always run with nightly builds.
Will the official version be released this year?

**Reply by timunie** [reply]
very likely, yes. As far as I see the biggest planned changes were done. Now it's time to sort out the biggest bugs before the final release.

**Comment by IsaacMarovitz** [comment, marked as answer]
Preview 5 is out. Thanks guys :)
